net/znc/Makefile

```diff
@@ -279,23 +279,17 @@ PKG_CONFIG_DEPENDS += $(patsubst %,CONFIG_PACKAGE_%,$(ZNC_MODULES))
 
 include $(INCLUDE_DIR)/package.mk
 
-CONFIGURE_VARS += \
-	CXXFLAGS="$(TARGET_CFLAGS) -fno-builtin" \
-	CPPFLAGS="-I$(STAGING_DIR)/usr/include -I$(STAGING_DIR)/include" \
-	LDFLAGS="-nodefaultlibs -lc -L$(STAGING_DIR)/usr/lib -L$(STAGING_DIR)/lib" \
-	LIBS="-lstdc++ -lm -lssl -lcrypto $(LIBGCC_S) -lc"
-
 CONFIGURE_ARGS += \
 	$(if $(CONFIG_ZNC_ICU), --enable-charset, --disable-charset) \
 	--disable-cyrus \
 	--disable-debug \
-	--enable-largefile \
 	--disable-perl \
-	--enable-poll \
 	--disable-python \
-	--enable-openssl \
 	--disable-swig \
 	--disable-tcl \
+	--enable-largefile \
+	--enable-poll \
+	--enable-openssl \
```

Comment:
Just curious, why did you reorder the options?
You may add a `--disable-optimization` to avoid appending `-O2` to `CFLAGS`, and save some real state.

Reply:
Grouped disable and enable ones together. It's a practice that I've seen with other packages.

Not sure what @KanjiMonster thinks of disabling O2.

Reply:
I'd prefer that we'd stick to the order of what ```./configure --help``` outputs whenever possible but that's me.